datadog_checks_base/datadog_checks/base/utils/fips.py

```diff
@@ -12,15 +15,15 @@ def enable_fips(path_to_openssl_conf=None, path_to_openssl_modules=None):
             path_to_embedded = _get_embedded_path() if path_to_embedded is None else path_to_embedded
             path_to_openssl_conf = path_to_embedded / "ssl" / "openssl.cnf"
             if not path_to_openssl_conf.exists():
-                raise RuntimeError(f'The configuration file "{path_to_openssl_conf}" does not exist')
+                LOGGER.warning('The configuration file "%s" does not exist', path_to_openssl_conf)
```

Comment:
Here and below: let's decide between `info` and `debug` here. Some questions to help decide:

- Who's the target audience here? Is it a user who's just interested if their agent works? Or is it a user with a broken agent or (Support) engineer who is helping this user?
- If the config file doesn't exist, can we enable FIPS mode? Looks like we can...